_pages/about/team.html

```diff
@@ -49,12 +49,7 @@ <h4 class="text-left animated hiding appear-animation-trigger" data-animation="f
             </h4>
             {% for teamMember in site.data.team-program %}{% include team-member.html teamMember=teamMember %}{% endfor %}
         </div>
-        <div id="academic-committee">
-            <h4 class="text-left animated hiding appear-animation-trigger" data-animation="fadeInUp" data-delay="0">
-                Academic Track committee
-            </h4>
-            {% for teamMember in site.data.team-academic %}{% include team-member.html teamMember=teamMember %}{% endfor %}
-        </div>
+
```

Comment:
I would add a link here to the academic committee page.

Reply:
Like it was before as text & link, or like a button link?

Reply:
As you prefer 😄